packages/int_datetime/lib/int_datetime.dart

```diff
@@ -15,14 +15,22 @@ int get dtNow => DateTime.now().millisecondsSinceEpoch;
 int get dtDay {
   final now = DateTime.now();
 
-  return DateTime.utc(now.year, now.month, now.day).millisecondsSinceEpoch;
+  return DateTime.utc(
```

Comment:
still [not clear](https://api.dart.dev/stable/2.10.1/dart-core/DateTime/DateTime.utc.html) how to use this constructor.

Reply:
It is temporary code I saved =)